index.html

```diff
@@ -208,6 +210,104 @@ <h3>GetUserMediaSemantics enum</h3>
+            {{boolean}}, defaulting to <code>true</code></dt>
+            <dd>See <a href=
+            "#def-constraint-avoidInefficientCompressedPixelFormat">
+            avoidInefficientCompressedPixelFormat</a> for details.</dd>
+          </dl>
+        </section>
+      </div>
+    </section>
+    <section id="mediatrackcapabilities-dictionary-extensions">
+      <h3>MediaTrackCapabilities dictionary extensions</h3>
+      <div>
+        <pre class="idl"
+>partial dictionary MediaTrackCapabilities {
+  sequence&lt;boolean&gt; avoidInefficientCompressedPixelFormat;
+};</pre>
+        <section>
+          <h2>Dictionary {{MediaTrackCapabilities}} Members</h2>
+          <dl data-link-for="MediaTrackCapabilities" data-dfn-for=
+          "MediaTrackCapabilities" class="dictionary-members">
+            <dt><dfn>avoidInefficientCompressedPixelFormat</dfn> of type
+            <code>sequence&lt;{{boolean}}&gt;</code></dt>
+            <dd>
+              <p>If the source has no inefficient compressed pixel formats, a
+              single <code>true</code> is reported. If the source only has
+              inefficient compressed pixel formats, a single <code>false</code>
+              is reported. If the script can control the feature, the source
+              reports a list with both <code>true</code> and <code>false</code>
+              as possible values. See <a href=
+              "#def-constraint-avoidInefficientCompressedPixelFormat">
+              avoidInefficientCompressedPixelFormat</a> for additional
+              details.</p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+    </section>
+    <section id="mediatracksettings-dictionary-extensions">
+      <h3>MediaTrackSettings dictionary extensions</h3>
+      <div>
+        <pre class="idl"
+>partial dictionary MediaTrackSettings {
+  boolean avoidInefficientCompressedPixelFormat;
+};</pre>
+        <section>
+          <h2>Dictionary {{MediaTrackSettings}} Members</h2>
+          <dl data-link-for="MediaTrackSettings" data-dfn-for=
+          "MediaTrackSettings" class="dictionary-members">
+            <dt><dfn>avoidInefficientCompressedPixelFormat</dfn> of type
+            {{boolean}}</dt>
+            <dd>See <a href=
+            "#def-constraint-avoidInefficientCompressedPixelFormat">
+            avoidInefficientCompressedPixelFormat</a> for details.</dd>
+      </section>
+      </div>
+    </section>
+      <h2>Constrainable Properties</h2>
+      <p>The constrainable properties in this document are defined below.</p>
+      <table class="simple">
+        <thead>
+          <tr>
+            <th>Property Name</th>
+            <th>Values</th>
+            <th>Notes</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><dfn id="def-constraint-avoidInefficientCompressedPixelFormat">
+            avoidInefficientCompressedPixelFormat</dfn></td>
```

Comment:
This will show up in `track.getSettings()`, which I like. But as a name, having a verb associated with this property doesn't make a lot of sense to me.

[Constrainable properties](https://w3c.github.io/mediacapture-main/#defining-a-new-constrainable-property) are usually not verbs (yes I'm aware I introduced "avoid"), but properties of the source that show up in constraints, capabilities _and_ settings. This makes me think that a better name would be `efficientCompressedPixelFormat` or `powerEfficient`. E.g.

```js
if (!track.getSettings().powerEfficient) {
  await track.applyConstraints({powerEfficient: true});
}
```
...or
```js
if (!track.getSettings().efficientCompressedPixelFormat) {
  await track.applyConstraints({efficientCompressedPixelFormat: true});
}
```

Reply:
Wait, that's not the correct opposite of `avoidInefficientCompressedPixelFormat`... `efficientOrUncompressedPixelFormat`? `efficientPixelFormat`? ugh.

Can we just go with `powerEfficient`?

Reply:
Fine by me. @youennf does your LGTM still hold if we call it powerEfficient?

Reply:
powerEfficient is a very broad term for camera. We might in the future want to add yet another powerEfficiency related property.
Given we nailed down the particular thing we want to constraint, I prefer more focused terms.

Also, we already have verb properties in getDisplayMedia so verbs are fine if they are more understandable.
I would stick with avoidInefficientCompressedPixelFormat, defaulting to false.
inefficientCompressedPixelFormat='include|exclude' might be okayish in the light of recent gDM preferences.

Reply:
I think "avoid" works both as a constraint (I would or would not like to avoid...) and as a setting (the current setting does or does not avoid). But this does feel very much like bike shedding. @jan-ivar

Reply:
powerEfficientPixelFormat is okayish too. The downsides is that a powerEfficientPixelFormat=false constraint seems to mean you want to select a non efficient pixel format while we want it to mean any pixel format is ok (distance is always the same). Maybe this is a bad idea though, when exposed as a setting you probably prefer to know whether the format is actually power efficient or not.

Reply:
powerEfficientPixelFormat which is true or false depending on whether or not it is efficient makes a lot of sense to me, I'd just have to remove the sentence that says that false is the same as the default. I agree the setting should reflect if the current setting is or is not efficient.

Reply:
Selecting a non-efficient pixel format seems useful for testing. It'd allow measuring the cost-benefit of forgoing these modes (which presumably have other decent qualities or they wouldn't exist).

Reply:
Okay I changed it to powerEfficientPixelFormat. Is this good enough to merge?

Reply:
Let's validate this during today's editor call.

Reply:
Is there a conclusion?

Reply:
It was brought that we should probably discuss this during a WebRTC interim.
This was discussed at yesterday's WebRTC interim meeting and there is consensus to move forward.
I am guessing we will be able to finalise this at next editor's call.

Reply:
OK, thank you :)